README.md

```diff
@@ -1,57 +1,250 @@
-Ballerina Twilio Connector
-===================
+# Ballerina Gmail Connector
  
 [![Build](https://travis-ci.org/ballerina-platform/module-ballerinax-twilio.svg?branch=master)](https://travis-ci.org/ballerina-platform/module-ballerinax-twilio)
 [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinax-twilio/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-twilio)
 ![GitHub release (latest by date including pre-releases)](https://img.shields.io/github/v/release/ballerina-platform/module-ballerinax-twilio?color=green&include_prereleases&label=latest%20release)
 [![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-twilio/actions/workflows/build-with-bal-test-native.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-twilio/actions/workflows/build-with-bal-test-native.yml)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-Twilio’s APIs power its platform for communications. 
+## Overview
 
-The Twilio connector allows you to send SMS, voice, and WhatsApp messages via the Twilio REST API. It also facilitates receiving inbound HTTP(S) requests (also known as webhooks) from Twilio's servers. 
- 
-For more information about configuration and operations, go to the module. 
-- [twilio](twilio/Module.md)
+The Twilio API provides the capability to access its platform for communications. These APIs connect the software layer and communication networks worldwide, enabling users to call and message anyone globally.
```

Comment:
Here, first we can add a single sentence to describe what twilio is and, another sentence to talk about what our Ballerina twilio connector supports right now.

Reply:
Sure. Is it ok in [here](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/209/commits/e53338ef5bd6bd3cc6640678b571a81183fbdbbf) ?

Reply:
+1